designer/client/data/section/addSection.ts

```diff
@@ -1,5 +1,10 @@
 import { FormDefinition, Section } from "@xgovformbuilder/model";
 
+/**
+ * @param data - Data from DataContext
```

Comment:
Unless 'Data' is referring to something written in code etc. with that case (eg uppercase 'D'), please change to the below - however, do we want to have 'DataContext'as `DataContext`?

* @param data - data from DataContext